### DISCUSSION TITLE
Slack looks very hostile. Is this a better place to talk?

### DISCUSSION THREAD

**Discussion by kissu** [teambit/bit#9032] [Community]
Hey! 👋🏻 
I joined Slack recently. But got no answers, just a deletion of my questions without any concrete discussion with anyone.
Now banned, with again no reason or interaction with a human. 🤷🏻‍♂️ 
I find it unprofessional and bizarre behavior, and I wondered if people here knew why.

Depending on the mood, I might ask my question here.

**Comment by davidfirst** [comment]
Hi @kissu — apologies on two counts: for the Slack experience, and for this thread sitting unanswered for so long.

I can't reconstruct what happened with your account back then, but being removed without a word from a human isn't how we want to treat someone showing up with a question.

The community Slack has since been retired — it wasn't being monitored, which is a large part of why your messages went nowhere. The community is on Discord now: https://discord.bit.cloud/. For bugs and feature requests, a GitHub issue here is the most reliable route, since those actually get tracked.

If the question you originally came with is still relevant, please ask it — here, in an issue, or on Discord — and you'll get an answer.
